src/components/verification-grid/verification-grid.ts

```diff
@@ -681,6 +814,12 @@ export class VerificationGridComponent extends WithShoelace(AbstractComponent(Li
     // related to: https://stackoverflow.com/a/78872316
     if (event.key === ESCAPE_KEY) {
       this.removeSubSelection();
+
+      // We reset the selection head so that if the user deselects all of the
+      // tiles (e.g. through the esc key), the next shift click will start a new
+      // range selection instead of starting from the old range selection
+      // position.
+      this.resetSelectionHead();
```

Comment:
do we need a preventDefault for escape?

Reply:
<img width="797" height="388" alt="image" src="https://github.com/user-attachments/assets/6ce00a89-ea2d-4e41-95fb-41f21b458588" />